client/src/components/checkbox/index.js

```diff
@@ -39,21 +49,30 @@ const Checkbox = ({ labelText, order, data, setData, objId, defaultValue }) => {
   }, [checkboxFocused]);
 
   const handleClick = () => {
-    setClicked(!clicked);
-    if (data && setData) {
-      let tempData = { ...data };
-      if (objId in tempData) {
-        if (clicked) {
-          // Removes Element
-          const index = tempData[objId].indexOf(labelText);
-          if (index >= -1) tempData[objId].splice(index, 1);
-          // Adds Element
-        } else tempData[objId].push(labelText);
-        setData(tempData);
-      } else {
-        console.error('objId not in the obj');
+    // For remember me checkbox
+    if(firebase){
+      rememberMe(!clicked) //pass the opposite because at the end of this function: setClicked(!clicked)
```

Comment:
I think it makes more sense to put the ```clicked``` state logic reversal here in the input of the function call rather than invert the rememberMe() if conditions.